lib/sdt_alloc.bpf.c

```diff
@@ -608,12 +624,21 @@ u64 scx_alloc_internal(struct scx_allocator *alloc)
 struct scx_static scx_static;
 
 __hidden
-void __arena *scx_static_alloc(size_t bytes)
+void __arena *scx_static_alloc(size_t bytes, size_t alignment)
 {
 	void __arena *memory, *old;
 	void __arena *ptr;
+	size_t padding;
+	u64 addr;
 
 	bpf_spin_lock(&alloc_lock);
+
+	/* Round up the current offset. */
+	addr = (__u64) scx_static.memory + scx_static.off;
+
+	padding = div_round_up(addr, alignment) - addr;
```

Comment:
Is this right? Shouldn't it be `div_round_up(addr, alignment) * alignment - addr`? Or rather `round_up(addr, alignment) - addr`?